config/environments/test.rb

```diff
@@ -47,6 +47,7 @@
     Bullet.unused_eager_loading_enable = false
     Bullet.counter_cache_enable = false
     Bullet.add_whitelist type: :n_plus_one_query, class_name: 'Audited::Audit', association: :user
+    Bullet.add_whitelist type: :n_plus_one_query, class_name: 'ApplicationChoice', association: :application_response_cache
```

Comment:
This seems a bit drastic, unless bullet also warns about writes/saves generating additional queries. If it's just the vendor api controllers/actions generating bullet warnings, perhaps we should be skipping bullet only for these specific actions, see https://github.com/flyerhzm/bullet#whitelist

Reply:
Yeah, it's not good is it? I'll revisit this once I've fixed the specs